x-pack/plugins/observability_solution/entity_manager/server/lib/entities/ingest_pipeline/generate_history_processors.ts

```diff
@@ -134,6 +134,17 @@ export function generateHistoryProcessors(definition: EntityDefinition) {
         method: 'MurmurHash3',
       },
     },
+    ...(definition.history.settings?.backfillSyncDelay
+      ? [
+          {
+            fingerprint: {
+              fields: ['entity.id', '@timestamp'],
```

Comment:
I wonder what implication this has given we could have collisions on entity.id with the way we compute it. Given we have two entities with the same id, we would only keep one since they would overwrite each other ?

Reply:
I guess same issue would happen on the `latest` transform since we merge on `entity.id` now, I suppose the likeliness of this happening is not a concern

Reply:
I will add the `entity.identity.*` fields and values as well which should distinguish them appropriately.